bin/i18n-update.sh

```diff
@@ -27,19 +27,19 @@ while test $# -gt 0; do
   case "$1" in
     -h|--help)
       echo "options:"
-      echo "-h, --help    show brief help"
-      echo "-p, --path    local path for generating files (by default a temp folder will be used)"
-      echo "-d, --debug   print extra info for debugging"
+      echo "-h, --help              show brief help"
+      echo "-p, --path              local path for generating files (by default a temp folder will be used)"
+      echo "-s, --skip-if-cache     skip update if cache is present (i.e. \"src/i18n-cache\" folder)"
```

Comment:
`skip-if-cache` option will allow us to only update the i18n localizations in case we're missing any of the cache folders of the plugins, this way we can attach it to dependency installation.

Reply:
This option is no longer required as the logic related to it has been introduced in [this script](https://github.com/wordpress-mobile/gutenberg-mobile/blob/40fa7390b95a6d6db2b148ccadff7abc6e62ad9e/bin/i18n-check-cache.sh).